### DISCUSSION TITLE
1.4.1-beta6

### DISCUSSION THREAD

**Discussion by ilyhalight** [ilyhalight/voice-over-translation#301] [Обновления]
⚠ Это бета версия и она может работать нестабильно. Обо всех проблемах рекомендуется сразу же сообщать в [Issues](https://github.com/ilyhalight/voice-over-translation/issues)

## Список изменений
- Добавлена поддержка ProxyTok (протестировано на proxitok.pabloferreiro.es и proxitok.privacydev.net. Ограничения теже, что и для обычного TikTok)
- Добавлена поддержка TikTok

  Работает только в видео типа: /@НИК/video/АЙДИ_ВИДЕО (не работает в ленте)

- Все селекторы сайтов добавленных в бета версиях были перенесены в selectors.js
- Исправлена ошибка из-за которой нельзя было выбрать языковую пару en-ru в mail.ru видео и rutube (#298)

**Full Changelog**: https://github.com/ilyhalight/voice-over-translation/compare/1.4.1-beta5...1.4.1-beta6

<hr /><em>This discussion was created from the release <a href='https://github.com/ilyhalight/voice-over-translation/releases/tag/1.4.1-beta6'>1.4.1-beta6</a>.</em>

**Comment by NullVerdict** [comment]
А что планируется в новой бете?

**Reply by ilyhalight** [reply]
Если будут какие-то новые проблемы в последней бете, то соответственно постараюсь исправить их. Так же, попробую добавить определение языка видео для остальных сайтов (vimeo, vk и т.д.). 

Дальше, либо подожду MrSoczekXD с его обновлением меню, логики загрузки и пр., либо если это займёт куда больше времени, то нужно будет самостоятельно пофиксить #285, #283, #282 и прочие, где кнопка не грузится из-за "@run-at document-start" и запуска main() после DOMContentLoaded (хотя, вроде бы, ты, тоже, это фиксил в своем PR).
